### ISSUE TITLE
buildStatic and Conditional Substitions with rollup enable in jspm@beta

### ISSUE THREAD

**Issue by fchiumeo** [systemjs/builder#549]
Hello
When I try to compile using conditions substitions and rollup enabled the javascript output is not generated successfully

**jspm.config.js**
````javascript
SystemJS.config({
    transpiler: "plugin-babel",
    babelOptions: {
        "plugins": [
            "babel-plugin-transform-decorators-legacy"
        ]
    }, 
    map: {
        "condiciones": "app/core/config/entorno.condiciones.js"
    }
});
````

**config.js**
```javascript
import "./config.#{condiciones|entorno}.js";

class OnConfig
{}
```
**entorno.condiciones.js**
````javascript
export var entorno = "desarrollo";
````

**bundle**
````javascript
 const outputOptions = {
        conditions: {
             "app/core/config/entorno.condiciones.js|entorno": entorno.toLocaleLowerCase()
        }
    };

    new jspm.Builder().buildStatic("app/main.js", "bundle.js",outputOptions)
        .then(() =>
        {
            log.info(chalk.blue("Generación de javascript completada con éxito."));
            callback();
        })
        .catch((ex) =>
        {
            log.error(chalk.red("Fallo la generación."));
            throw ex;
        });
````
**bundle.js**
````javascript
(["1","@dummy-entry-point"], [], function($__System) {

......

$__System.register("@dummy-entry-point", ["3", "2", "4"], function($__export) {
  "use strict";

.....

````

*console error*
**`uncaught exception: Module 1 not present.`**

disable rollup en outputOptions(`rollup: false`) bundle.js is ok

**bundle.js**
````javascript
(["1","2"], [], function($__System) {

......

$__System.register("1", ["4", "3", "5", "7"], function($__export) {
  "use strict";

.....

$__System.register("2", [], function($__export) {
  "use strict";
.....
````

I'm using jspm 0.17.0-beta.13

**Comment by guybedford** [comment]
This may have been fixed in https://github.com/systemjs/builder/commit/104ff4f52f92ee1ade3c8ef91e9207e58c337f95.

**Comment by fchiumeo** [comment]
not working yet :cry:

**Comment by guybedford** [comment]
I can get this to work fine in jspm 0.17.0-beta.13 with:

```
jspm build app/main.js bundle.js --conditions "{'app/core/config/entorno.conditiones.js|entorno':'value'}"
```

Let me know if you're still having issues here.

**Comment by fchiumeo** [comment]
same error, not working with command and gulp task

**Comment by guybedford** [comment]
@fchiumeo can you perhaps share a full replication then? Also please confirm you're definitely running the latest beta.

**Comment by fchiumeo** [comment]
[fchiumeo/app-jspm](https://github.com/fchiumeo/app-jspm)
i'm testing on jspm 0.17 beta 13 and systemjs/builder#master

**Comment by guybedford** [comment]
Ok thanks, this actually has nothing to do with the conditional side of things, it's a bug with the rollup optimization process. I'm looking into it.

**Comment by guybedford** [comment]
Thanks for the test case, fixed in https://github.com/systemjs/builder/commit/548879c269f4da96772b35c831902a1c58a70b55.

**Comment by fchiumeo** [comment]
now working fine
very thanks

**Comment by guybedford** [comment]
Released in 0.15.16.
